alchemist-utils.el

```diff
@@ -138,6 +138,15 @@ For example, convert 'my_app/my_module.ex' to 'MyApp.MyModule'."
       (format "%s/" path)
     path))
 
+(defun alchemist-utils--regex-in-buffer-p (buffer regex)
+  "Return non-nil if the given `BUFFER' contains at least one occurrence of
+  `REGEX', nil otherwise."
+  (with-current-buffer buffer
+    (save-excursion
+      (save-match-data
+        (beginning-of-buffer)
```

Comment:
We should use `(goto-char (point-min))` instead the interactive function `(beginning-of-buffer)`

Part from the `beginning-of-buffer` description:

> ...
> Don't use this command in Lisp programs!
> (goto-char (point-min)) is faster.

If you would byte-compile the `alchemist-utils.el` file you would get something like the following:

> :Warning: `beginning-of-buffer' is for interactive use only;
>  use `(goto-char (point-min))' instead.